prep-css-layout-classes/index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>CSS Layout Classes</title>
+  <link rel="stylesheet" href="reset.css">
+  <style>
+    /* stylelint-disable block-no-empty */
+    body {
+      font-family: sans-serif;
+    }
+
+    h1 {
+      text-align: center;
+    }
+    h3 {
+      text-align:center;
+    }
+    p {
+      text-align:center;
+    }
+
+    .container {
+      max-width: 992px;
+      margin: auto;
+    }
+
+    .row {
+      width: 100%;
+      display: flex;
+
+    }
+
+    .column {
+      flex-basis: calc(100%/3);
+      padding-left: 8px;
+      padding-right: 8px;
+    }
+    img {
+      width: 100%;
+    }
+  </style>
+</head>
+<body>
+  <h1> Indiana Jones </h1>
+  <div class="container">
+    <div class="row">
+      <div class="column"><img src="images/raiders.jpg"></div>
+      <div class="column"><img src="images/temple.jpg"></div>
+      <div class="column"><img src="images/crusade.jpg"></div>
+    </div>
+  </div>
+  <div class="container">
+    <div class="row">
+      <h3 class="column"> Raiders of the Lost Ark</h3>
+      <h3 class="column">Temple of Doom</h3>
+      <h3 class="column">The Last Crusade</h3>
+    </div>
+  </div>
+  <div class="container">
+    <div class="row">
+      <p class="column">Lucas conceived Raiders of the Lost Ark in the early 1970s. Seeking to modernize the serial films of the early 20th century, he developed the idea further with Kaufman, who suggested the Ark as the film's goal.  Lucas eventually focused on developing his 1977 space opera Star Wars</p>
+      <p class="column">The film was released to financial success but initial reviews were mixed, criticizing its darker elements and violence. However, critical opinion has improved since 1984, citing the film's intensity and imagination.</p>
+      <p class="column"> After the mixed reaction to Indiana Jones and the Temple of Doom, Spielberg chose to tone down the dark tone and graphic violence in the next installment.  During the five years between The Temple of Doom and The Last Crusade, he and executive producer Lucas reviewed several scripts before accepting Jeffrey Boams's.</p>
+    </div>
```

Comment:
Hi Lean, try reorganizing your HTML to only 1 container div, 1 row div, and 3 column divs. Each column should contain an img, an h3, and a p element